ui/src/main/java/org/jtaccuino/core/ui/extensions/DisplayExtension.java

```diff
@@ -130,241 +113,62 @@ public void display(Object object, Consumer<Integer> counterConsumer) {
-                        mouseOldY = mousePosY;
-                        mousePosX = me.getSceneX();
-                        mousePosY = me.getSceneY();
-                        mouseDeltaX = (mousePosX - mouseOldX);
-                        mouseDeltaY = (mousePosY - mouseOldY);
-
-                        double modifier = 10.0;
-                        double modifierFactor = 0.1;
-
-                        if (me.isControlDown()) {
-                            modifier = 0.1;
-                        }
-                        if (me.isShiftDown()) {
-                            modifier = 50.0;
-                        }
-                        if (me.isPrimaryButtonDown()) {
-                            ry.setAngle(((ry.getAngle() + mouseDeltaX * modifierFactor * modifier * 2.0) % 360 + 540) % 360 - 180);  // +
-                            rx.setAngle(((rx.getAngle() - mouseDeltaY * modifierFactor * modifier * 2.0) % 360 + 540) % 360 - 180);  // -
-                        } else if (me.isSecondaryButtonDown()) {
-                            double z = camera.getTranslateZ();
-                            double newZ = z + mouseDeltaX * modifierFactor * modifier;
-                            camera.setTranslateZ(newZ);
-                        } else if (me.isMiddleButtonDown()) {
-                            t.setX(t.getX() + mouseDeltaX * modifierFactor * modifier * 0.3);  // -
-                            t.setY(t.getY() + mouseDeltaY * modifierFactor * modifier * 0.3);  // -
-                        }
-                    });
-
-                    lastEffect = System.nanoTime();
-                    AtomicInteger count=new AtomicInteger();
-                    AnimationTimer timerEffect = new AnimationTimer() {
-                        @Override
-                        public void handle(long now) {
-                            if (now > lastEffect + 500_000_000l) {
-                                counterConsumer.accept(count.getAndIncrement());
-                                lastEffect = now;
+                .orElseThrow()
+                .get()
+                .render(object)
+                .orElseGet(() -> {
+                    var label = new Label("Automatic conversion for type " + object.getClass() + " + to javafx.scene.Node failed somehow!");
+                    return label;
+                });
+        if (null != counterConsumer) {
+            node.parentProperty().addListener(new InvalidationListener() {
+                @Override
+                public void invalidated(Observable observable) {
+                    if (node.getParent() != null) {
+                        lastEffect = System.nanoTime();
+                        AtomicInteger count = new AtomicInteger();
+                        AnimationTimer timerEffect = new AnimationTimer() {
+                            @Override
+                            public void handle(long now) {
+                                if (now > lastEffect + 500_000_000l) {
```

Comment:
nitpick: for better optical discernability
```suggestion
                                if (now > lastEffect + 500_000_000L) {
```